atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -69,6 +69,17 @@ def run(self):
         self.log.debug('getting next release from build info: %s', build_info)
         next_release = self.xmlrpc.getNextRelease(build_info)
 
+        # getNextRelease will return the release of the last successful build
+        # but next_release might be a failed build
```

Comment:
We could also mention that `CGImport` doesn't allow importing a build whose nvr matches that of a failed build, since that's the reason we have to do all this.